mkdocs.yml

```diff
@@ -4,10 +4,11 @@ repo_url: https://github.com/smaranjitghose/awesome-portfolio-websites
 nav:
     - Home: index.md
     - Getting Started: gs.md
```

Comment:
- Home
- Getting Started
- Changing Contents (Restructure into another md file after the cloning guide like I did for img-ai-boilerplate
- Checking Locally
- Pushing your Portfolio to GitHub 
- Hosting

** Do this in the next couple of PRs